components/brave_wallet_ui/components/desktop/views/portfolio/components/accounts-and-transctions-list/index.tsx

```diff
@@ -108,10 +108,7 @@ export const AccountsAndTransactionsList = ({
   }, [accounts, selectedAsset])
 
   const accountsList = React.useMemo(() => {
-    if (isNonFungibleToken && selectedAssetsNetwork) {
-      return filteredAccountsByCoinType.filter((account) => Number(account.nativeBalanceRegistry[selectedAssetsNetwork.chainId] ?? 0) !== 0)
-    }
-    return filteredAccountsByCoinType
+    return filteredAccountsByCoinType.filter((account) => getBalance(account, selectedAsset) > '0')
```

Comment:
Wondering if numeric comparison for strings is considered safe. I think `Amount(balance).gt(0)` is a better option since it covers a wide range of edge cases.